API.md

```
@@ -326,6 +387,62 @@ curl -H "Authorization: Bearer YOUR_TOKEN" \
+```
+
+---
+
+### Get Container Storage History
+
+Retrieve historical writable-layer and volume sizes for a container.
+
+**Endpoint:** `GET /api/container/:containerId/disk/history`
+
+**Query Parameters:**
+| Parameter | Type | Required | Default | Description |
```

markdownlint-cli2 (0.23.2)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)